src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -58,26 +59,13 @@
 			<div class="tw-w-full tw-my-4">
 				<kv-pagination
 					v-show="!emptyState"
-					:total="totalCount >= 12 ? 12 : totalCount"
+					:total="totalCount"
 					:limit="loanSearchState.pageLimit"
 					:offset="loanSearchState.pageOffset"
 					@page-changed="pageChange"
 					:scroll-to-top="false"
 				/>
 			</div>
-			<div v-show="showSeeMoreCta" class="tw-w-full tw-my-4 tw-text-center">
-				<kv-button
-					variant="secondary"
-					:href="filterPageUrl()"
```

Comment:
Looks like `filterPageUrl` can be removed?

Reply:
TY